### PR TITLE
My nick has change since then, also changed the website position so the first one is the more known one

### PR DESCRIPTION
**Pull request by Zerlingg**

### CODE REVIEW

**Comment by Zerlingg** [comment]
> update version

wdym? its literally the one in here just that i changed my name and the url position

**Comment by Bas950** [comment]
> > update version
> 
> wdym? its literally the one in here just that i changed my name and the url position

if you dont update the version it wont change it

**Comment by Zerlingg** [comment]
> > > update version
> > 
> > 
> > wdym? its literally the one in here just that i changed my name and the url position
> 
> if you dont update the version it wont change it

wdym with update, update what

**Comment by Bas950** [comment]
> > > > update version
> > > 
> > > 
> > > wdym? its literally the one in here just that i changed my name and the url position
> > 
> > 
> > if you dont update the version it wont change it
> 
> wdym with update, update what

bump the version (make it 1 higher)

File: websites/A/AnimeFLV/dist/metadata.json [hunk 1]
```diff
@@ -15,8 +15,8 @@
     "zh_CN": "AnimeFLV是一个视频网站，您可以在其中观看带有西班牙语字幕的动漫。"
   },
   "url": [
-    "animeflv.ru",
-    "animeflv.net"
+    "animeflv.net",
+    "animeflv.ru"
   ],
   "version": "1.1.6",
```
Comment (Kotoki1337) [inline websites/A/AnimeFLV/dist/metadata.json:21]:
```suggestion
  "version": "1.1.7",
```
if you do not change the version, the system will not think that you have changed file/source code, which means that your merge will not be applied

**Comment by Bas950** [comment]
Hasn't made required changes within a week. - Closed

**Pull request closed without merging**